src/oop/object_independence/medium/q2/TestWaitList.java

```diff
@@ -0,0 +1,11 @@
+package oop.object_independence.medium.q2;
+
+public class TestWaitList {
+    public static void main(String[] args) {
+        PhoneWaitList wlA = new PhoneWaitList(5);
+        PhoneWaitList wlB = new PhoneWaitList(2);
```

Comment:
You should also get the student to modify one of the wait lists by calling the set method because the main point of this category is to get the students to recognize that different objects have different internal values.